.github/pull_request_template.md

```diff
@@ -1,24 +1,66 @@
-<!--- Provide a summary of your changes in the title field above. -->
+<!--- Provide a summary of your changes in the title field above. For guidance on formatting, see the comment the bottom of this template. -->
+
+# Description
 
-## Description
 <!--- Describe your changes. -->
 
 ## Motivation & context
+
 <!--- What problem does this change solve? -->
 <!--- Provide a link if you are addressing an open issue. -->
 
 ## Issue type checklist
-<!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+<!--- What type of change are you submitting? Put an x in boxes that apply: -->
+
+[] **Chore**: A change that does not impact distributed packages.
+[] **Bug fix**: A change which fixes an issue, link to the issue above.
+[] **New feature**: A change that adds functionality.
+
+**Is this a breaking change?**
+
+<!--- Is this a change that will cause current functionality to break? If yes, put an x in the "yes" box: -->
+
+[] Yes
+
+<!--- If yes, describe the impact to FAST-DNA in detail. -->
```

Comment:
do we need to specify FAST-DNA here? It seems unecessary. Prefer:

"if yes, describe the impact."

Reply:
Is this really a blocking issue?